src/synapse/azext_synapse/commands.py

```diff
@@ -0,0 +1,93 @@
+
+
+# pylint: disable=too-many-statements
+def load_command_table(self, _):
+    from azure.cli.core.commands import CliCommandType
+    from ._client_factory import cf_synapse_client_workspace_factory
+    from ._client_factory import cf_synapse_client_bigdatapool_factory
+    from ._client_factory import cf_synapse_client_sqlpool_factory
+    from ._client_factory import cf_synapse_spark_batch
+    from ._client_factory import cf_synapse_spark_session
+
+    synapse_workspace_sdk = CliCommandType(
+        operations_tmpl='azext_synapse.vendored_sdks.azure_mgmt_synapse.operations#WorkspacesOperations.{}',
+        client_factory=cf_synapse_client_workspace_factory)
+
+    synapse_bigdatapool_sdk = CliCommandType(
+        operations_tmpl='azext_synapse.vendored_sdks.azure_mgmt_synapse.operations#BigDataPoolsOperations.{}',
+        client_factory=cf_synapse_client_bigdatapool_factory)
+
+    synapse_sqlpool_sdk = CliCommandType(
+        operations_tmpl='azext_synapse.vendored_sdks.azure_mgmt_synapse.operations#SqlPoolsOperations.{}',
+        client_factory=cf_synapse_client_sqlpool_factory)
+
+    synapse_spark_batch_sdk = CliCommandType(
+        operations_tmpl='azext_synapse.vendored_sdks.azure_synapse.operations#SparkBatchOperations.{}',
+        client_factory=cf_synapse_spark_batch
+    )
+
+    synapse_spark_session_sdk = CliCommandType(
+        operations_tmpl='azext_synapse.vendored_sdks.azure_synapse.operations#SparkSessionOperations.{}',
+        client_factory=cf_synapse_spark_session
+    )
+
+    # Management Plane Commands
+    with self.command_group('synapse workspace', synapse_workspace_sdk,
+                            client_factory=cf_synapse_client_workspace_factory) as g:
+        g.show_command('show', 'get')
+        g.custom_command('list', 'list_workspaces')
+        g.custom_command('create', 'create_workspace', supports_no_wait=True)
+        g.custom_command('update', 'update_workspace', supports_no_wait=True)
+        g.command('delete', 'delete', confirmation=True, supports_no_wait=True)
+        g.wait_command('wait')
+
+    with self.command_group('synapse spark pool', synapse_bigdatapool_sdk,
+                            client_factory=cf_synapse_client_bigdatapool_factory) as g:
+        g.custom_command('show', 'get_spark_pool')
+        g.command('list', 'list_by_workspace')
+        g.custom_command('create', 'create_spark_pool', supports_no_wait=True)
+        g.custom_command('update', 'update_spark_pool', supports_no_wait=True)
+        g.custom_command('delete', 'delete_spark_pool', confirmation=True, supports_no_wait=True)
+        g.wait_command('wait')
+
+    with self.command_group('synapse sql pool', synapse_sqlpool_sdk,
+                            client_factory=cf_synapse_client_sqlpool_factory) as g:
+        g.show_command('show', 'get')
+        g.command('list', 'list_by_workspace')
+        g.custom_command('create', 'create_sql_pool', supports_no_wait=True)

```

Comment:
If you have supports_no_wait=True, you should have a wait command.

Reply:
> If you have supports_no_wait=True, you should have a wait command.

We have

Reply:
Please check below command groups as well.